i18n/en.json

```
@@ -42,23 +42,23 @@
 		},
 		"data-users-section": {
 			"title": "We provide the data, you provide the ideas",
 			"subtitle": "For data consumers",
-			"description": "<p>On the one hand, the internet offers many ways to find out about Berlin’s diverse cultural scene, but on the other hand, there is still a lack of a central overview of all the city’s cultural institutions and events. With kulturdaten.berlin, we want to create the basis for making it easier to discover the diversity of Berlin’s culture - from nice, small events in the neighborhood to major cultural productions. Via our data interface, event portals, for example, can obtain exactly the data they need for their particular sector focus in no time at all.</p><p>kulturdaten.berlin is based on the principles of open data and open source. That means, the source code is open and the cultural data of the platform is freely and easily accessible to all via an API, e.g. to use it for own applications. Whether it’s an app specifically for barrier-free cultural experiences or the expansion of a neighborhood website with concert tips in the neighborhood: we open up Berlin’s cultural data for subsequent use to anyone with fresh ideas.</p>",
+			"description": "<p>On the one hand, the internet offers many ways to find out about Berlin’s diverse cultural scene, but on the other hand, there is still a lack of a central overview of all the city’s cultural institutions and events. With kulturdaten.berlin, we want to create the basis for making it easier to discover the diversity of Berlin’s culture - from nice, small events in the neighborhood to major cultural productions. Via our data interface, event portals, for example, can obtain exactly the data they need for their particular sector focus in no time at all.</p><p>kulturdaten.berlin is based on the principles of open data and open source. That means, the source code is open and the cultural data of the platform is freely and easily accessible to all via an API, e.g. to use it for own applications. Whether it’s an app specifically for barrier-free cultural experiences or a website with concert tips in the neighborhood: we open up Berlin’s cultural data for subsequent use to anyone with fresh ideas.</p>",
 			"use-case-calendar-title": "Use Case: Cultural Calendar",
-			"use-case-calendar-description": "<p>Does your calendar provide a broad overview, cater specifically to children, or curate the most exciting events for literature enthusiasts?</p><p>Export or filter data – For example, by target audience or cultural genre.</p>",
+			"use-case-calendar-description": "<p>Does your calendar provide a broad overview, cater specifically to children, or curate the most exciting events for literature enthusiasts?</p><p>Export or filter data e.g. by target audience, location or cultural genre.</p>",
```

Here we should add a comma before "e.g."!

```suggestion
			"use-case-calendar-description": "<p>Does your calendar provide a broad overview, cater specifically to children, or curate the most exciting events for literature enthusiasts?</p><p>Export or filter data, e.g. by target audience, location or cultural genre.</p>",
```